.github/workflows/write-prerelease-comment.yml

```diff
@@ -90,24 +90,14 @@ jobs:
             npm install https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-package-cloudflare-pages-shared-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }}
             ```
 
+            ```sh
+            npm install https://prerelease-registry.devprod.cloudflare.dev/workers-sdk/runs/${{ env.WORKFLOW_RUN_ID_FOR_WRANGLER }}/npm-package-create-cloudflare-${{ env.WORKFLOW_RUN_PR_FOR_WRANGLER }}
```

Comment:
Note: this is using `env.WORKFLOW_RUN_ID_FOR_WRANGLER` and `env.WORKFLOW_RUN_PR_FOR_WRANGLER` (not the C3 specific env variable) for right or wrong this is consistent with all the other packages in this comment